database/articles.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const express = require('express');
```

Comment:
Middleware is usually only in your `server.js` and added as application level middleware so that it would be available to all routes. Also we don't invoke `express` here we invoke `express.Router` like this:

```
var express = require('express')
var router = express.Router()
```